src/plugins/tracking/activity.js

```diff
@@ -84,23 +83,23 @@ const onRequest = (request, reply) => {
  * tracking cookies that need to be set using request.response.state.
  */
 const getOnPreResponse = cookieConfig => (request, reply) => {
-	const { sessionId, trackId } = request.plugins[ACTIVITY_PLUGIN_NAME];
-	const { sessionIdCookieName, trackIdCookieName, isProd } = cookieConfig;
-	const cookieOpts: CookieOpts = {
+	const { browserIdCookieName, trackIdCookieName } = cookieConfig;
+	const pluginData = request.plugins[ACTIVITY_PLUGIN_NAME];
+	const browserId = pluginData.browserIdCookieName;
+	const trackId = pluginData.trackIdCookieName;
+
+	const FOREVER: CookieOpts = {
 		encoding: 'none',
 		path: '/',
 		isHttpOnly: true,
-		isSecure: isProd,
```

Comment:
`isSecure` is the default, and is now supported dev and prod, so we can leave it out